controllers/mysqlcluster_controller_test.go

```diff
@@ -703,6 +703,9 @@ var _ = Describe("MySQLCluster reconciler", func() {
 		Expect(sts.Spec.Template.Spec.SecurityContext).NotTo(BeNil())
 		Expect(*sts.Spec.Template.Spec.SecurityContext.FSGroup).To(Equal(int64(constants.ContainerGID)))
 		Expect(*sts.Spec.Template.Spec.SecurityContext.FSGroupChangePolicy).To(Equal(corev1.FSGroupChangeOnRootMismatch))
+		Expect(sts.Spec.Template.Spec.Affinity).NotTo(BeNil())
+		Expect(sts.Spec.Template.Spec.Affinity.PodAntiAffinity).NotTo(BeNil())
+		Expect(sts.Spec.Template.Spec.Affinity.PodAntiAffinity.PreferredDuringSchedulingIgnoredDuringExecution).NotTo(BeNil())
 
```

Comment:
Could you add a test case when the affinity is set in the MySQLCluster?

Reply:
I changed it to test the affinities set by the user:
https://github.com/cybozu-go/moco/pull/513/commits/9d0667a8100028f94e9398f3ac2e52f46c0cc988